src/lib/replication/repl_dev/raft_repl_dev.cpp

```diff
@@ -39,7 +39,10 @@ RaftReplDev::RaftReplDev(RaftReplService& svc, superblk< raft_repl_dev_superblk
         m_data_journal = std::make_shared< ReplLogStore >(
             *this, *m_state_machine, m_rd_sb->logdev_id, m_rd_sb->logstore_id,
             [this](logstore_seq_num_t lsn, log_buffer buf, void* key) { on_log_found(lsn, buf, key); },
-            [this](std::shared_ptr< HomeLogStore > hs, logstore_seq_num_t lsn) { m_log_store_replay_done = true; });
+            [this](std::shared_ptr< HomeLogStore > hs, logstore_seq_num_t lsn) {
+                m_log_store_replay_done = true;
+                set_log_store_last_durable_lsn(hs->tail_lsn());
```

Comment:
this line is the actual fix , which will set m_last_durable_lsn to the tail_lsn(last lsn) of log store after all the logs are replayed. @xiaoxichen